src/tests/ftest/pool/multiple_creates_test.py

```diff
@@ -60,7 +60,9 @@ def test_create_one(self):
         """
         Test issuing a single  pool create commands at once.
 
-        :avocado: tags=all,pool,smoke,pr,daily_regression,small,createone
+        :avocado: tags=all,weekly_regression
```

Comment:
Typo?
```suggestion
        :avocado: tags=all,full_regression
```

Reply:
please fix in a subsequent PR.

Reply:
Haha.  _weekly_.  Will fix in whatever I push next.